staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -240,6 +240,16 @@ var (
 		[]string{"source", "status"},
 	)
 
+	//clientRequestCounter is a number of total requests with client name
+	clientRequestCounter = compbasemetrics.NewCounterVec(
```

Comment:
We already have a metric for request counter:
https://github.com/kubernetes/kubernetes/pull/106216/files#diff-c4f07ee44a4626c817dd95cf30e147a2125ea923804be5c3608fb4a845fc1c20R74

We used to have a client label there, but that was actually a vulnerability, as the client is a potentially unbounded dimensions.

So we don't want to add that.

@logicalhan - FYI

Reply:
That makes sense.  We will try to look out operator which do not use listwatch by other method.

Thanks!

Reply:
Yeah we can't do this... client is unbounded and can be manipulated in the request; it is therefore an attack vector.

Reply:
ok，thanks